docusaurus.config.js

```diff
@@ -71,6 +71,14 @@ module.exports = {
           activeBasePath: 'docs/js',
           sdkSwitch: true
         },
+        {
+          label: '🐍 Python',
+          to: '/',
+          position: 'left',
+          docsPluginId: '',
+          activeBasePath: '/',
+          sdkSwitch: true
```

Comment:
This should lead to a (fake?) python page, that redirects to the python docs

Reply:
removed